web/src/main/java/com/zebrunner/reporting/web/SocialController.java

```diff
@@ -18,11 +20,17 @@
 @ConditionalOnProperty(name = "twitter.enabled", havingValue = "true")
 public class SocialController extends AbstractController implements SocialDocumentedController {
 
+    private static final String ZEBRUNNER_NEWS = "zebrunner_news";
+
     private final TwitterService twitterService;
 
     @GetMapping("/twitter/timeline")
     @Override
-    public List<Tweet> getUserTimeline(@RequestParam("userName") String userName, @RequestParam(value = "pageSize", defaultValue = "20") int pageSize) {
+    public List<Tweet> getUserTimeline(@RequestParam(value = "userName", defaultValue = ZEBRUNNER_NEWS) String userName, @RequestParam(value = "pageSize", defaultValue = "20") int pageSize) {
```

Comment:
Let's use `v1/social` as a controller-level mapping and `tweets` as a method level mapping. That way you can extend this controller with other social integrations in future. All new APIs should be versioned ideally. api part will be resolved at a gateway level (we already have this configuration in place)